nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md

```diff
@@ -7,7 +7,7 @@ component: Core
 
 ## Automatic subscriptions
 
-The default mode for managing subscriptions is "auto subscribe". When a subscriber endpoint is started it determines to which events it needs to subscribe. It then sends subscription messages to the [owning endpoint](/nservicebus/messaging/message-owner.md) for those messages.
+The default mode for managing subscriptions is "auto subscribe". When a subscriber endpoint is started it determines to which events it needs to subscribe. It then sends subscription messages to the publisher endpoint for those messages.
```

Comment:
not related directly to this change, but should the list starting with line 16 also contain a point that message driven pub/sub also requires registered routes for autosubscribe to work?